proposals/eligibility_validator.go

```diff
@@ -140,7 +142,8 @@ func (v *Validator) CheckEligibility(ctx context.Context, ballot *types.Ballot)
 		beacon,
 	)
 
-	v.beacons.ReportBeaconFromBallot(epoch, ballot.ID(), beacon, weight)
+	weightPer := fixed.DivUint64(atxWeight, uint64(numEligibleSlots))
```

Comment:
this number was already computed somewhere in big.Rat, as it is basically weight / slots

Reply:
are you talking about here?

https://github.com/spacemeshos/go-spacemesh/blob/e9d1a564bda11964b9a49f3d820757cd7989a140/proposals/util/util.go#L45

i decided to do the calculation here (instead of calling the library) because all the info (atx weight, active set and numEligibleSlots) was already calculated earlier in this function.